jekyll/_cci2/triggers-overview.adoc

```diff
@@ -64,6 +64,31 @@ curl -X POST https://circleci.com/api/v2/project/<VCS>/<your-vcs-org-or-username
 
 TIP: You can also specify pipeline parameters when triggering a pipeline using the API. For full details, see the https://circleci.com/docs/api/v2/index.html#operation/triggerPipeline[API v2 Docs].
 
+For **projects integrated with GitHub App or Bitbucket Data Center**, you can trigger a pipeline for a project using the endpoint `https://circleci.com/api/v2/project/{project-slug}/pipeline/run`. 
+
+NOTE: This endpoint is currently in preview, and is not yet documented in the link:https://circleci.com/docs/api/v2/index.html[official API V2 documentation].
+
+. If you have not already, get set up to use API v2 by following the steps in the  <<api-developers-guide#authentication-and-authorization,API Developers Guide>>. Here you will find instructions on how to find your project slug.
+
+. To retrieve the **Definition ID** for the **Pipeline** you want to run, navigate to your project's **Project Settings**, and select **Pipelines** in the sidebar. In the column **Definition ID**, select btn:[Copy] for the correct Pipeline. 
```

Comment:
@rosieyohannan I wonder if we need an image to illustrate this?

Reply:
Yep I'll add a screenshot. Can I get access to the preview?